ERRROR compiling example SimpleOnButton

Compiling your simple example I receive this error

c:\Arduino2\libraries\OneButton\src\OneButton.cpp: In constructor 'OneButton::OneButton(int, bool, bool)':
c:\Arduino2\libraries\OneButton\src\OneButton.cpp:33:7: error: too many arguments to function 'void init()'
   33 |   init(pin, pullupActive ? INPUT_PULLUP : INPUT, activeLow);
      |   ~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from c:\Arduino2\libraries\OneButton\src\OneButton.h:28,
                 from c:\Arduino2\libraries\OneButton\src\OneButton.cpp:19:

--- a/src/OneButton.cpp
+++ b/src/OneButton.cpp
@@ -30,7 +30,7 @@ OneButton::OneButton() {
 
 // Initialize the OneButton library.
 OneButton::OneButton(const int pin, const bool activeLow, const bool pullupActive) {
-  init(pin, pullupActive ? INPUT_PULLUP : INPUT, activeLow);
+  setup(pin, pullupActive ? INPUT_PULLUP : INPUT, activeLow);
 }  // OneButton
 
 

